powerauth-push-server/src/main/java/io/getlime/push/controller/rest/InboxController.java

```diff
@@ -77,34 +77,23 @@ public ObjectResponse<GetInboxMessageCountResponse> fetchMessageCountForUser(
         return new ObjectResponse<>(inboxService.fetchMessageCountForUser(userId, appId));
     }
 
-    @GetMapping("users/{userId}/messages/{id}")
-    public ObjectResponse<GetInboxMessageDetailResponse> fetchMessageDetail(
+    @PutMapping("users/{userId}/read-all")
+    public Response readAllMessages(
             @NotNull @Size(min = 1, max = 255) @PathVariable("userId") String userId,
-            @NotNull @Size(min = 1, max = 255) @RequestParam("appId") String appId,
-            @NotNull @PathVariable("id") String inboxId) throws InboxMessageNotFoundException, AppNotFoundException {
-        return new ObjectResponse<>(inboxService.fetchMessageDetail(userId, appId, inboxId));
+            @NotNull @Size(min = 1, max = 255) @RequestParam("appId") String appId) throws AppNotFoundException {
```

Comment:
Shouldn't we be consistent? I see `appId` here but `applications` at `users/{userId}`. What about to have `appIds` there?

Reply:
In this case, it does not make much sense. Generally speaking, using just `appId` would suffice for the current states. The reason we have `applications` in the list is that for some admin interfaces (in `powerauth-cloud`), I can imagine requesting messages for multiple applications.